source/objects/ptoSimClass.m

```diff
@@ -103,6 +103,13 @@
             'rho',                              'NOT DEFINED',...           % Fluid density
             'k1',                               'NOT DEFINED',...           % Valve coefficiente
             'k2',                               'NOT DEFINED')              % Valve coefficient, it's a function of the other valve variables
+        simpleDirectDrivePTO (1,1) struct      = struct(...                 % simple direct drive linear PTO Block properties
+            'Kt',                               'NOT DEFINED',...           % Torque constant
+            'Ng',                               'NOT DEFINED',...           % Gear ratio
+            'J',                                'NOT DEFINED',...           % Drivetrain inertia
```

Comment:
I suggest to change the names of these variables for something more comprehensive. Instead of "J", something like "drivetrainInertia", instead of "B" something like "drivetrainFriction", etc.